com.unity.render-pipelines.universal/ShaderLibrary/UnityGBuffer.hlsl

```diff
@@ -159,9 +159,9 @@ BRDFData BRDFDataFromGbuffer(half4 gbuffer0, half4 gbuffer1, half4 gbuffer2)
     half smoothness = gbuffer2.a * 0.5h + 0.5h;
 #endif
 
-    BRDFData brdfData;
+    BRDFData brdfData = (BRDFData)0;
     half alpha = 1.0; // NOTE: alpha can get modfied, forward writes it out (_ALPHAPREMULTIPLY_ON).
-    InitializeBRDFDataDirect(gbuffer0.rgb, gbuffer1.rgb, reflectivity, oneMinusReflectivity, smoothness, alpha, brdfData);
+    InitializeBRDFDataDirect(diffuse, specular, reflectivity, oneMinusReflectivity, smoothness, alpha, brdfData);
```

Comment:
Ok, I misread the code, no problem.

Reply:
I just reused the local variables already there. I thought it would be more consistent to have all the vars named in the function call and in case someone modifies the local vars. They should be eliminated by the compiler.